src/References.php

```diff
@@ -40,7 +40,12 @@ public function __construct()
 
         $config = require $referenceFile;
 
-        $keysToCheck = ['colors', 'interlace_types'];
+        $keysToCheck = [
+            'colors',
+            'colorspace_values',
+            'interlace_types',
+            'paper_sizes',
+        ];
         $keysExist = true;
```

Comment:
OK, I was working on implementing the references too, but I had forgotten this...

Reply:
Np, sometimes, more brains are useful 😉